test/common/utils/accessible-menus.e2e.spec.js

```diff
@@ -0,0 +1,228 @@
+    client.expect.element('#vetnav-healthcare').to.not.be.visible;
+    client
+      .focusOn(secondMenuItem)
+      .keys(RIGHT_ARROW)
+      .assert.visible('#vetnav-healthcare');
+    client.assert.isActiveElement(hcFirstItem);
+
+    // Space should open the sub-menu and focus on the first element if sub-menu exists
+    client.expect.element('#vetnav-disability').to.not.be.visible;
+    client
+      .focusOn(firstMenuItem)
+      .keys(SPACE)
+      .assert.visible('#vetnav-disability');
+    // TODO: Uncomment this when the code that makes this work is merged in
+    // client.assert.isActiveElement(disFirstItem);
+
+    // Enter should open the sub-menu and focus on the first element if sub-menu exists
+    client.expect.element('#vetnav-healthcare').to.not.be.visible;
+    client
+      .focusOn(secondMenuItem)
+      .keys(ENTER)
+      .assert.visible('#vetnav-healthcare');
+    // TODO: Uncomment this when the code that makes this work is merged in
+    // client.assert.isActiveElement(hcFirstItem);
+
+
+    // -- Non-sub menu buttons -- //
+    const prescriptionsLink = 'a[href="/health-care/prescriptions"]';
+
+    client.click(benefitsButton);
+
+    // Right arrow should do nothing if no sub-menu exists
+    client
+      .focusOn(prescriptionsLink)
+      .keys(RIGHT_ARROW)
+      .assert.visible(prescriptionsLink);
+
+    // TODO: Write these; I'm not sure what to do with them
+    //  Do we just poke them and test the url? Seems like it'd get pretty slow
+    // Space should follow the link if the menu item is a link
+    // Enter should follow the link if the menu item is a link
+
+    client.click(exploreButton); // Open our explore menu back up again
+
+
+    // ---------------------- //
+    // --- Sub-menu tests --- //
+    // ---------------------- //
+
+    client.click(secondMenuItem); // Open up the health care menu
+
+    // Down arrow should move focus to the next item
+    client
+      .focusOn(hcFirstItem)
+      .keys(DOWN_ARROW)
+      .assert.isActiveElement(hcSecondItem);
+    //  with wraparound
+    client
+      .focusOn(hcLastItem)
+      .keys(DOWN_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+
+    // Up arrow should move focus to the previous item
+    client
+      .focusOn(hcSecondItem)
+      .keys(UP_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+    //  with wraparound
+    client
+      .focusOn(hcFirstItem)
+      .keys(UP_ARROW)
+      .assert.isActiveElement(hcLastItem);
+
+    // Right arrow should do nothing
+    client
+      .focusOn(hcFirstItem)
+      .keys(RIGHT_ARROW)
+      .assert.isActiveElement(hcFirstItem);
+
+    // Left arrow should move focus back to the opening menu button
+    client
+      .focusOn(hcFirstItem)
+      .keys(LEFT_ARROW)
+      .assert.isActiveElement(secondMenuItem);
+
+    // TODO: Write these; I'm not sure what to do with them
+    //  Do we just poke them and test the url? Seems like it'd get pretty slow
```

Comment:
It makes sense not to test this here, and to emphasize focus management and whether things are opening/closing as expected.

Reply:
🤔  Yeah, I guess this is the same as clicking, which is outside the scope of keyboard navigation, isn't it?

What do y'all think?